website/static/js/fangorn.js

```diff
@@ -754,6 +754,11 @@ function _fangornDropzoneSuccess(treebeard, file, response) {
     }
     var url = item.data.nodeUrl + 'files/' + item.data.provider + item.data.path;
     addFileStatus(treebeard, file, true, '', url);
+
+    if (item.data.provider === "dataverse") {
```

Comment:
This and below will make `jshint` fail with the message `Strings must use singlequote`.